src/eterna/rscript/ROPHighlight.ts

```diff
@@ -64,7 +64,7 @@ export default class ROPHighlight extends RScriptOp {
                 log.warn(`ROPHighlight: missing highlight parent [id='${this._uiElementString}']`);
                 return;
             }
-            // if (highlightParent instanceof PIXI.DisplayObject) {
+            // if (highlightParent instanceof DisplayObject) {
```

Comment:
I'm not sure why this is here in the first place, to be honest - didn't have the time to investigate (so didn't want to make assumptions to remove it), but also wanted to update in case it should eventually be uncommented

Reply:
Got it! Nothing else on my end to query. Feel free to do that thing!